# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Version.inl
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/Version.inl
--- a/gfx/angle/checkout/src/libANGLE/Version.inl	Sat Sep 07 13:36:46 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,59 +0,0 @@
-//
-// Copyright (c) 2015 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-
-// Version.inl: Encapsulation of a GL version.
-
-#include <tuple>
-
-namespace gl
-{
-
-constexpr Version::Version()
-    : Version(0, 0)
-{
-}
-
-// Avoid conflicts with linux system defines
-#undef major
-#undef minor
-
-constexpr Version::Version(unsigned int major_, unsigned int minor_)
-    : major(major_),
-      minor(minor_)
-{
-}
-
-inline bool operator==(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) == std::tie(b.major, b.minor);
-}
-
-inline bool operator!=(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) != std::tie(b.major, b.minor);
-}
-
-inline bool operator>=(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) >= std::tie(b.major, b.minor);
-}
-
-inline bool operator<=(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) <= std::tie(b.major, b.minor);
-}
-
-inline bool operator<(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) < std::tie(b.major, b.minor);
-}
-
-inline bool operator>(const Version &a, const Version &b)
-{
-    return std::tie(a.major, a.minor) > std::tie(b.major, b.minor);
-}
-
-}  // namespace gl